tools/python3/phloping.py

```
@@ -124,20 +122,67 @@ def normalised_times_for_L(self, ilvl):
+
+def print_root_as_csv(scope_timer_file, n_parts, headers=None, regex=None):
+    stf = scope_timer_file  # alias
+    stf = file_parser(stf) if isinstance(stf, str) else stf
+
+    if headers:
+        print(",".join(headers))
+    for root in stf.roots:
+        s = stf(root.k)
+        if regex and regex not in s:
+            continue
+        bits = s.split(",")
```

Ruff

Local variable `bits` is assigned to but never used

Remove assignment to unused variable `bits`

(F841)